v2/emailpassword/graphql-integration/backend-setup.mdx

```diff
@@ -32,69 +36,187 @@ Follow the [frontend and backend custom UI setup guides](../custom-ui/init/front
 
 ## b) Install GraphQL dependencies on your backend
 
-:::note
-We use express-graphql as an example, SuperTokens can be used with other GraphQL libraries.
+:::info
+We use Apollo server in this example, SuperTokens can be used with other GraphQL libraries as well.
 :::
 
 ```bash
-npm i express-graphql graphql
+npm install @apollo/server graphql
 ```
 
 ## c) Setup GraphQL
 
-### Create a schema 
+### Create your schema 
 
-```tsx
-import { buildSchema } from 'graphql';
+In this example we will create a simple social app that allows logged in users to create and view posts. To build an app like this you would need the following:
+- A way for users to login and logout
+- A way for users to create posts
+- A way for users to like posts
 
-var schema = buildSchema(`
-  type Query {
-    hello: String
-  }
-`);
+For sake of simplicity we will just focus on login/logout and viewing posts.
+
+From a GraphQL perspective we will need the following:
+- A query to fetch the user's posts
+
+```tsx
+const typeDefs = `
+type Query {
+    posts: [String],
+}
+`;
 ```
 
+The `posts` Query will return a list of `String` (one for each post) if the user is logged in.
+
+:::important
+We do not add any queries or mutations for login and logout because the SuperTokens middleware exposes APIs that handle this for us.
+:::
+
+<PreBuiltOrCustomUISwitcher>
+
+<PreBuiltUIContent>
+
+Refer to the [backend pre-built UI setup guides](../pre-built-ui/setup/backend) if you have not already added the SuperTokens middleware.
```

Comment:
there is a `:::` rendered after this line in the preview.. please be more careful

Reply:
Right, the fix https://github.com/supertokens/docs/pull/606/files/c08df8b05ba0650c2c315aac0e7eaea0c92af80b#diff-2ec77ed993af09a1c71927dccd687f8475db697bd5c70d7fddb1d8eb74e6aabb actually broke this. The trailing `:::` is before the Create resolvers section and is intentional